h2o-py/h2o/model/metrics_base.py

```diff
@@ -134,15 +134,11 @@ def show(self):
             print("Misclassification Error (Categorical): " + str(self.cat_err()))
 
     def r2(self):
-        """
-        :return: Retrieve the R^2 coefficient for this set of metrics
-        """
+        """The R^2 coefficient."""
```

Comment:
In Python docstrings are governed by [PEP-257](https://www.python.org/dev/peps/pep-0257/). It dictates rather stringent rules, such as "each docstring should start with a 1-line summary that should be a full sentence (ending with a dot) in imperative mood". So a doc-string with just the ":return:" part is not valid, and triggers warning from the linter.

Now, the reason why I make such change only in these two cases is because I'm too lazy to fix EVERYTHING at once, and prefer doing small fixes here-and-there wherever they catch my eye :)